manifests/init.pp

```diff
@@ -325,9 +329,11 @@
       register_in_foreman   => $register_in_foreman,
       foreman_base_url      => $foreman_url,
       registered_proxy_url  => "https://${capsule_fqdn}:${capsule::foreman_proxy_port}",
+      trusted_hosts         => undef,
```

Comment:
Because https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/params.pp#L38 and http://projects.theforeman.org/issues/9102